types/hooks/screens/useChatUtilsHook.types.ts

```diff
@@ -9,7 +14,7 @@ interface UseChatUtilsHook {
   goToMoveChat: (channel: ChannelList) => void;
   goToPostDetailScreen: (channel: ChannelList) => void;
   goToCommunityScreen: (channel: ChannelList) => void;
-  goToContactScreen: () => void;
+  goToContactScreen: ({from}: ContactScreenPayload) => void;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new function `goToContactScreen` now takes a parameter of type `ContactScreenPayload`. This is a good change as it allows the function to know whether the user is anonymous or signed in. However, it's important to ensure that all calls to this function are updated to pass this parameter.
```diff
-  goToContactScreen: () => void;
+  goToContactScreen: ({from}: ContactScreenPayload) => void;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->